doc/antora-playbook.yml

```diff
@@ -0,0 +1,20 @@
+# partial playbook just for this project to be able to compile documentation without pushing it to GitHub
+# usage: antora generate antora-playbook.yml
+site:
+  title: SmallRye Reactive Messaging only documentation
+  start_page: smallrye-reactive-messaging::index.adoc
+content:
+  sources:
+    - url: ..
+      start_path: doc
+      branches: HEAD
+ui:
+  bundle:
+    url: https://github.com/smallrye/smallrye-antora-ui/blob/master/build/ui-bundle.zip?raw=true
+    snapshot: true
+asciidoc:
```

Comment:
unfortunately we have to include variables in the playbook for now, Antora 2.2 doesn't support them in component descriptor (`antora.yml`) so the same set of variables will need to be for the main playbook (within `smallrye.github.io` repository) for the time being